docs/shared-content/concepts/create-projects.include.md

```diff
@@ -6,3 +6,21 @@ Projects are used to collect all the assets that make up your deployment process
 
 1. Navigate to the **Projects** tab, and click **ADD PROJECT**.
 1. Give the project a name, for instance, *Hello, world*, and click **Save**.
+
+**Optional**
+
+By default, Octopus Deploy will store the deployment process, runbook process, and variables in the back-end SQL Server.  Starting with **Octopus 2022.1**, you have the option to store that information in a git repository.  
```

Comment:
I think changing this to state upfront what is stored will provide better clarity to customers.

```suggestion
By default, Octopus Deploy will store the deployment process, runbook process, and variables in the back-end SQL Server.  From **Octopus 2022.1**, you have the option to store the *deployment process* in a git repository.  

:::hint
The ability to store runbook processes and variables will be added in future versions of Octopus.
:::
```

Reply:
I agree. I would also add the following.

From **Octopus 2022.1**, you have the option to store the *deployment process* in a git repository, and we're adding support for variables and runbook processes in the future.